HIP-0013.md

```diff
@@ -0,0 +1,130 @@
+# HIP-0013 : Standard for Profile Data via TXT Records 
+
+```
+Number:  HIP-0013
+Title:   Standard for Profile Data via TXT Records 
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal to create a standard for the labels of string attributes in TXT records for profile data — following the [RFC 1464](https://datatracker.ietf.org/doc/html/rfc1464) standard. Parts of this standard where inspired by [EIP-634](https://eips.ethereum.org/EIPS/eip-634).
+
+## Motivation
+
+Handshake domains can also be used as usernames. Domains are used to sign in to websites and to identify users in the Handshake ecosystem. Therefore it's useful to be able to attach profile information like an email, phone, avatar or service to a Domain.
+
+This Standard can also be used to provide contact information for domain listings according to [HIP-0010](https://github.com/handshake-org/HIPs/blob/master/HIP-0010.md).
+
+## TXT Record Standard: Profile Data
+
+For the above to work we need standards so any website or app can properly render the TXT records. The records should be flexible, short and easy to understand. It's important to note, that all values should be checked by the application before serving them to the user.
+
+The general syntax for profile data is:
+
+**TXT** `profile <attribute name>=<attribute value>`
+
+There are currently five supported attributes: **[email](#email)**, **[phone](#phone)**, **[avatar](#avatar)** and **[service](#service)**. All attributes are optional but at least one attribute needs to be present, for a Domain to count as a profile.
```

Comment:
What are your thoughts on adding "name" to this list? Could be an individual's name or a company's.